test/arrayops.jl

```diff
@@ -1869,7 +1869,7 @@ end
     end
 end
 
-isdefined(:TestHelpers) || include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))
+isdefined(:TestHelpers) || eval(Main, :(include(joinpath(dirname(@__FILE__), "TestHelpers.jl"))))
```

Comment:
Shouldn't `include("TestHelpers.jl")`  be sufficient?  `include` is already relative to the path of the current file by default.